dear_petition/petition/models.py

```diff
@@ -339,13 +337,51 @@ def get_all_offense_records(self, filter_active=True, include_annotations=True):
         return qs
 
     def has_attachments(self):
-        return self.attachments.count() > 0
+        return self.documents.count() > 1
 
 
 # Look-alike Petition object used to support JSON data-driven petitions
 DataPetition = namedtuple("DataPetition", ["form_type", "data_only"], defaults=[True])
 
 
+class PetitionDocument(models.Model):
```

Comment:
What does `PetitionDocument` provide over `Petition`? `Petition` provides attachments and linked offense records (now through `PetitionOffenseRecord`).

Reply:
I updated the description to explain.